pages/docs/getting-started/quickstart.mdx

```diff
@@ -183,4 +183,4 @@ oasTools.initialize(app).then(() => {
     http.createServer(app).listen(serverPort, () => console.log("Server started!"));
 })
 ```
-Once everything is done, run `node index` and open [your new server docs](http://localhost:8080/docs) in your browser. You may check the [Project setup](../fundamentals/project-setup) for more advanced options when setting up your project.
\ No newline at end of file
+Once everything is done, run `node index` and open [your new server docs](http://localhost:8080/docs) in your browser. You may check the [Project setup](../fundamentals/project-setup) for more advanced options when setting up your project.
```

Comment:
Hey the `Project setup` link is wrong, maybe you can fix this also while you have a PR open

```suggestion
Once everything is done, run `node index` and open [your new server docs](http://localhost:8080/docs) in your browser. You may check the [Project setup](../customization/project-setup) for more advanced options when setting up your project.
```